### ISSUE TITLE
Honor .dockerignore file

### ISSUE THREAD

**Issue by Ri0n** [edrevo/dockerfile-plus#12]
Hi
I'm writing a rust application and start it both in a container and locally.
I added to .dockerignore file to not transfer `target` (local) into docker context and it seems to work well regular docker build, but with Dockerfile-plus it seems to transfer 5Gb context anyway.
Of course I can workaround this by explicitly setting the target directory for my local builds but it's a little inconvenient.

**Comment by jedzill4** [comment]
Same issue here, and I'm working on an ML project with +100gb of data

**Comment by ghost** [comment]
Yes, using INCLUDE+ the .dockerignore is ignored when transferring context. This causes massive build delays when there's large data in the repository or when using a .venv.
